src/Integration.UnitTests/Suppression/SuppressedIssuesProviderTests.cs

```diff
@@ -49,6 +53,45 @@ public void TestInitialize()
             testSubject = new SuppressedIssuesProvider(activeSolutionBoundTracker, createProviderFunc.Object);
         }
 
+        [TestMethod]
+        public void MefCtor_TwoExports_Singleton()
+        {
+            // SuppressIssuesProvider is unusual as it exports two types via MEF.
+            // We expect both types to be exportable, and for them to return the
+            // same instance.
```

Comment:
The comment explains why we can't use the normal MEF test pattern.

I added an overload to `MefTestHelpers` to simplify writing this test and ones like it.